nelmio/cors-bundle/1.5/config/packages/nelmio_cors.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+return static function (RoutingConfigurator $routingConfigurator): void {
+};
```

Comment:
This is on `dev-main` not correctly converted now but was correctly converted on released version:

![Bildschirmfoto 2022-07-28 um 12 46 56](https://user-images.githubusercontent.com/1698337/181487949-1f85aea5-357f-4330-bd64-60092d5510d2.png)

Reply:
Could your narrow down to breaking commit of `config-transformer`?

Reply:
Is also fixed. Thank you!